### PR TITLE
DDCE-4413 Add Armed Forces

### PR DESCRIPTION
**Pull request by gastonracui**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -260,6 +260,7 @@ immigration.dependant_aop_other='Afghan Operation Pitting (other)'
 immigration.dependant_aop_other_lotr='Afghan Operation Pitting (other)'
 immigration.ukraine_scheme='Ukraine Scheme'
 immigration.global_business_mobility='Global Business Mobility'
+immigration.armed_forces='Armed Forces Settlement'
```
Comment (Taiwo-AB) [inline conf/messages:263]: Is there anywhere to confirm that this updated content is valid? Although it reads better than the one in the spreadsheet, can't find the approval of this updated content anywhere.

Comment (gastonracui) [reply conf/messages:263]: we sent them an email, dont think that route should be part of the content for the user. On the official document they showed like this

Screenshot 2023-07-19 at 12 30 21

Comment (gastonracui) [reply conf/messages:263]: but there is no other information

Comment (Taiwo-AB) [reply conf/messages:263]: Yh I can see it [here](https://www.gov.uk/apply-indefinite-leave-to-remain-armed-forces) on gov.uk but feel we should get approval from someone first. Will get approval from Lorenzo.

Comment (Taiwo-AB) [reply conf/messages:263]: Obtained approval and added comment on JIRA with new content. Will add screenshots of testing this change with dynamic stub feature.

**Review by Taiwo-AB — APPROVED** [review]

**Pull request merged**